recipe/meta.yaml

```
@@ -8,22 +8,23 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sf-hamilton-{{ version }}.tar.gz
-  sha256: 91f71fa81e37efb156ae8901c4b7fc073f794de9ebe7ff8a59db5943de52c0eb
+  sha256: 9dda6ed13c829cedda02eda0713f17995ea23a8511f7a32df61d9744c249ee60
 
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True #[py<36]
 
 requirements:
   host:
     - pip
-    - python >=3.7,<4
+    - python
+    - wheel
```

```suggestion
    - setuptools
    - wheel
```
Because `setuptools` is implicitly a _python build tool_ of this package, see https://github.com/DAGWorks-Inc/hamilton/blob/8bbe58f71e78d6142deb3c635d0960a589b84ead/setup.py#L7. But we should say it explicitly

---

1. Any particular reason we have to list implicit dependencies for Python? 
2. Do we list implicit dependencies for other packages?

---

1. implicit in mean that it does not exists in https://github.com/DAGWorks-Inc/hamilton/blob/8bbe58f71e78d6142deb3c635d0960a589b84ead/requirements.txt
but rather directly in setup.py as import statement, would fail if env does not contain setuptools
2. yes, making them explicit which is usually better